script.js

```diff
@@ -0,0 +1,139 @@
+$(document).ready(function(){
+  var num=1,
+      aleat,
+      btnClicked,
+      typeofGame,
+      allColors=['green','red','blue','yellow'],
+      rightSequence=[],
+      personSequence=[],
+      order=0,
+      storeColInf ={
+        "red": 1,
+        "green": 2,
+        "blue": 3,
+        "yellow": 4   
+      };
+  
+  changeColorDisplay('none');
+  $('.starter').attr('disabled','disabled');
+  
+  $('.starter').on("click", function(){
+    typeofGame = $(this).attr('id')
+    if (typeofGame=='strict'){
+      $("#strictActive").addClass('strictActive');
+    }
+    $('.starter').attr('disabled','disabled');
+    continueGame();
+  });  
+
+  $(".color").on("click", function(){
+    var idClicked = $(this).attr('id');
+    order++;
+    lightSound(''+idClicked+'');
+    personSequence.push(idClicked);
+    verify(''+idClicked+'');  
+  });
+
+  $("#number").on("errou", function(event, callback){
+    $("#number").html("--");
+    setTimeout(function(){
+      callback();
+      $("#number").html(num);
+    },1500);
+  });
+
+  $('.offBtn').on('click', function(){
```

Comment:
you are repeating this string through the code, it would be good if you set it up in a variable